### DISCUSSION TITLE
LongPress and Tap command on one control simultaneously

### DISCUSSION THREAD

**Discussion by QuadMan** [CommunityToolkit/Maui#2276] [Q&A]
I have Tap command and LongPressCommand on the same label. 
`                <Label Text="Test">
                    <Label.GestureRecognizers>
                        <TapGestureRecognizer Command="{Binding TapCommand}"/>
                    </Label.GestureRecognizers>
                    <Label.Behaviors>
                        <toolkit:TouchBehavior          
                            LongPressDuration="1000"
                            LongPressCommand="{Binding LongPressCommand}"/>
                    </Label.Behaviors>
                </Label>`

And some times just two clicks (with small delay) on label leads to longpress command execution. Is there a way to prevent it?
PS. I tried to make Command in TouchBehavior - the same results.

**Comment by ghost** [comment]
Thank you @QuadMan , I have found wierd behaviour on my side too. [I have issued a new issue](https://github.com/CommunityToolkit/Maui/issues/2894).
